invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/TinyMCEEditor.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Editor } from "@tinymce/tinymce-react";
+import "tinymce/tinymce";
+import "tinymce/models/dom/model";
+import "tinymce/themes/silver";
+import "tinymce/icons/default";
+import "tinymce/skins/ui/oxide/skin.min.css";
+import "tinymce/plugins/table";
+import "tinymce/plugins/autoresize";
+import "tinymce/plugins/codesample";
+import "tinymce/plugins/link";
+import "tinymce/plugins/lists";
+
+export class TinyMCEEditor extends Component {
+  handleEditorBlur = (event, editor) => {
+    const content = editor.getContent();
+    const modifiedContent = `<div class="mce-content">${content}</div>`;
+    this.props.onBlur(modifiedContent);
+  };
+
+  render() {
+    const { initialValue } = this.props;
+
+    return (
+      <Editor
```

Comment:
could we move this component globally to react-invenio-forms module and import it in the needed packages? that would allow to re-use the configuration globally as well